docker/ol-docker-start.sh

```diff
@@ -3,6 +3,17 @@
 # quick method to start all ol services from one script
 # inside an container, bypass all upstart/services
 
+CONFIG=conf/openlibrary-docker.yml
+
+reindex-solr() {
```

Comment:
Agree, I actually tried to look into doing this, but didn't think I could do it quickly without risking breaking all the other make tasks. I decided we already had a big limitation that the Makefile assumed hardcoded solr locations for tasks and that correcting it would be a full feature. I also think that most of that Makefile needs a review to check we even want to have all of those tasks there.   I have strong doubts about more than half of them, they are either vagrant only, not used, or possibly broken, but I'm not sure :)

I feel there should be an independent "review Makefile" issue, and more discussion. I also wasn't sure if putting functions with loops in Makefiles was nice. Perhaps the reindex task should be a script that is called from the Makefile?  (I know you're not a fan of :all-the-things: in '/scripts' but I'm not sure the Makefile is right for it either ;) )

Reply:
The makefile definitely needs some love :P if it's more work than a) add env variable to docker-compose, b) add env variable to vagrant, c) replace hardcoded config file in Makefile with said variable, then I wouldn't worry about it. Certainly not mission critical, and it'll be easy to clean up once we replace vagrant.

(Also, the current implementation in the Makefile doesn't use loops, just has to (repetitive) commands. Actually, since this uses a regex to query psql, it could be done on just one command...)

Reply:
oh wow, that's heaps better in one command :man_facepalming:  I thought my loop was better than the duplication, not really. I'll revisit!

Reply:
Oh, drats, I'm wrong; I think maybe their order matters (because one gets queried for the other) :/

Reply:
I think I'll leave it as is for now, if that's ok?